### PR TITLE
Expose list of ongoing video compressions and improved logging

### PR DESCRIPTION
**Pull request by rodja**
As requested in https://github.com/zauberzeug/field_friend/pull/155#issuecomment-2315503366, this PR adds logging to the video compression so we can see when it finishes. It also introduces the `ongoing_computations` property which holds a list of all target video filenames which are currently created.

### CODE REVIEW

File: rosys/analysis/timelapse_recorder.py [hunk 1]
```diff
@@ -70,8 +72,12 @@ async def save(self, image: RosysImage) -> None:
                                   self._notifications.pop(0) if self._notifications else [],
                                   self.frame_info_builder(image))
 
-    async def compress_video(self) -> None:
-        """Creates a video from the captured images"""
+    def compress_video(self) -> Task:
+        """Creates a video from the captured images
+        Note: this method starts a background task and returns immediately."""
```
Comment (falkoschindler) [inline rosys/analysis/timelapse_recorder.py:77]:
```suggestion
        """Create a video from the captured images.
        
        Note: This method starts a background task and returns immediately.
        """
```
Following PEP257 for multi-line docstrings.

File: rosys/analysis/timelapse_recorder.py [hunk 2]
```diff
@@ -45,6 +46,7 @@ def __init__(self, *, width: int = 800, height: int = 600, capture_rate: float =
         self.last_capture_time = rosys.time()
         self._notifications: list[list[str]] = []
         self.camera: Camera | None = None
+        self.ongoing_computations: list[str] = []
```
Comment (falkoschindler) [inline rosys/analysis/timelapse_recorder.py:49]: `ongoing_compressions` might be a slightly more expressive name.

**Review by falkoschindler — APPROVED** [review]
@rodja I reviewed the pull request once again as a whole. Should we still wait for @pascalzauberzeug, or can we merge into main? This is currently the last open ticket for version 0.16.0.

**Comment by pascalzauberzeug** [comment]
@falkoschindler I could test it on F13 later today, but i'm fine with just merging it

**Comment by rodja** [comment]
Me too.lets merge

**Pull request merged**